src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -25,7 +25,7 @@ def _get_vm_version(vm_type):
 
 class BackupTests(ScenarioTest, unittest.TestCase):
     @ResourceGroupPreparer(location="southeastasia")
-    @VaultPreparer()
+    @VaultPreparer(soft_delete=False)
```

Comment:
Why disable soft delete?

Reply:
The tests were failing otherwise as it was not able to clean-up the vault. This was the error it was showing.
![image](https://user-images.githubusercontent.com/26112239/123038170-84a4fa80-d40d-11eb-9368-ad085e838964.png)

So I disabled soft-delete wherever it was not required.

Reply:
OK